src/main/java/uk/gov/hmcts/reform/em/npa/domain/MarkUp.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.em.npa.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedBy;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.util.UUID;
+
+/**
+ * A MarkUp.
+ */
+@Entity
+@Table(name = "markup")
+@Getter
+@Setter
+public class MarkUp implements Serializable {
+
+    @Id
+    private UUID id;
```

Comment:
We won't need this? we can use a composite primary key. See the Tag entity in Annotations API

Reply:
Discussed and agreed to leave this as it is.